env0/resource_aws_credentials.go

```diff
@@ -62,29 +63,32 @@ func resourceAwsCredentials() *schema.Resource {
 }
 
 func resourceAwsCredentialsCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	_, accessKeyExist := d.GetOk("access_key_id")
```

Comment:
Enforcing the exactlyOnce in the code level only for creation (not for read/import).